assets/config.php

```diff
@@ -33,6 +33,7 @@
         Stancl\Tenancy\Bootstrappers\FilesystemTenancyBootstrapper::class,
         Stancl\Tenancy\Bootstrappers\QueueTenancyBootstrapper::class,
         // Stancl\Tenancy\Bootstrappers\RedisTenancyBootstrapper::class, // Note: phpredis is needed
+        // Stancl\Tenancy\Bootstrappers\BatchTenancyBootstrapper::class,
```

Comment:
@Riley19280 Could this be enabled by default? Would that lead to any issues?

Reply:
> Could this be enabled by default? Would that lead to any issues?

I think we can enable it by default, and there will be no issue. I enabled it by default, and all tests passed since it's switching batch connection only when tenancy is initialized. 

My only concern is whether we need to enable it by default if the app is not using BatchJobs. It's similar to `RedisTenancyBootstrapper`. We don't need it until we are using Redis.

Reply:
> We don't need it until we are using Redis

No — it's not a matter of *need*. Redis is disabled because *it depends on a specific environment (phpredis)*.

If the Batch tenancy bootstrapper won't cause any issues by being enabled by default, then we can keep it enabled.

Reply:
Seems like my reply to this didn't get submitted either. I wrote that Redis is unique because **there's an environment dependency (phpredis)**, that's why it's disabled by default.

We don't really care about which parts of Laravel the developer is using. We just disable Redis by default because the package will run into exceptions if the developer isn't using phpredis.

Reply:
I see. 

But my point is still valid. 
> My only concern is whether we need to enable it by default if the app is not using BatchJobs.

Not every app will need batching. If the developer uses batch, he will enable this bootstrapper otherwise, keeping it disabled is okay.

Reply:
Sure, but not every app will need queues either. Yet we keep it enabled by default. I think the bootstrapper **doesn't do anything unless it's used**. Therefore it can simply be enabled by default. Again, the only reason why we'd even disable any bootstrapper in the first place — what we did with Redis — is environment requirements that would make the app fail.

Reply:
>  is environment requirements that would make the app fail.

Make sense. 

So going to enable the bootstrapper by default.